starknet_py/net/schemas/rpc.py

```diff
@@ -117,7 +124,7 @@ def make_dataclass(self, data, **kwargs):
 
 
 class TransactionSchema(Schema):
-    hash = Felt(data_key="transaction_hash", required=True)
+    hash = Felt(data_key="transaction_hash", load_default=None)
```

Comment:
Is it because the transaction_hash moved outside of TXN in the spec?

Reply:
Exactly, in case `TXN` is returned without `transaction_hash` None is loaded by default